docs/agent/sensor-full-disk-access.asciidoc

```diff
@@ -0,0 +1,40 @@
+[[sensor-full-disk-access]]
+== Enable full disk access
+
+Elastic cloud users can take advantage of Endgame's detection sensor. However, after installing the sensor on either macOS 10.14 Mojave or 10.15 Catalina, you must enable a setting called **Full Disk
+Access** so the sensor can analyze files that are in your home directory.
+
+[float]
+=== Access to the Endgame sensor
+
+Review the https://www.elastic.co/support_policy/endpoint-system-requirements[Elastic Endpoint Security System Requirements] and then https://support.elastic.co/customers/s/login/[contact Elastic support] to gain access to the Endgame legacy sensor. This sensor is only available to current Elastic and Endgame customers.
+
+
+[float]
```

Comment:
Remove. This is likely an endpoint user clicking. They wont be able to "gain access" to the sensor.